index.html

```diff
@@ -30,10 +37,12 @@ <h2>I'm Mariane <br> Glad to see you!</h2>
     </section>
     <section class="works">
```

Comment:
Great design @vflorin1995 🤝

- [ ] As specified in the project requirements, kindly stick to the design provided in the Figma template as much as possible. Kindly fix the color of your `technologies` as well as `See Project` buttons.
- Your design
![Your tech](https://user-images.githubusercontent.com/63917725/186623695-7aadb235-44d3-46a9-b1ad-c7a676ffa97e.PNG)
- Figma design
![Figma tech](https://user-images.githubusercontent.com/63917725/186623748-02ac8c89-c99d-4af1-8b20-a71837ce5632.PNG)